content/hardware/04.pro/boards/portenta-x8/tutorials/01.user-manual/content.md

```diff
@@ -1,1440 +1,1363 @@
+
+Once connected, you will see the Portenta X8 LEDs start blinking. Portenta X8 features two LEDs, a Power LED, and a Status LED, which can blink in parallel.
+
+![Portenta X8 LEDs](assets/portenta_x8_leds.png "Portenta X8 LEDs")
+
+The table below describes the meaning and functionalities of LEDs.
+
+| **LED Type** | **Colour** | **Meaning**                                     |
+|--------------|------------|-------------------------------------------------|
+| Power LED    | Red        | Power ON                                        |
+| Status LED   | White      | OS booting in progress                          |
+| Status LED   | Blue       | Linux Kernel running                            |
+| Status LED   | Green      | Board connected to the Internet                 |
+| Status LED   | Red        | STM32H7 LED, blinking when triggered in the IDE |
+
+### Setup with the Arduino Linux Wizard
+
+***It is recommended that you have your Portenta X8 with the latest OS version. Check [this section](#portenta-x8-os-image-update) to learn how to have your Portenta X8 up-to-date.***
+
+Once the Portenta X8 is correctly powered up, you can start interacting with it. 
+
+Open your browser and navigate to [www.arduino.cc/start](www.arduino.cc/start). The following page will be displayed.
+
+![First Setup](assets/registration-homepage.png "First Setup")
+
+Click on **Linux boards** and sign in to your Arduino Cloud account. If you do not have an account, create a new one from the same webpage.
+
+![Sign into Arduino Cloud](assets/registration-signin.png "Sign into Arduino Cloud")
+
+When successfully logged in, you will be asked to download the **Arduino Create Agent** if you have not done so yet. Click on **DOWNLOAD**.
+
+![Create Agent Installation](assets/agent-installation.png "Create Agent Installation")
+
+The agent will be installed on your computer. This activity might take a few minutes. Once installed, your Portenta X8 will be automatically detected by your PC after a few seconds.
+
+![Portenta X8 successfully detected](assets/board-detection.png "Portenta X8 successfully detected")
+
+Now click on **START CONFIGURATION**. The tool will install all the required add-ons to make your Portenta X8 work efficiently with your PC via serial communication.
+
+You can now proceed to the setup of the board connectivity by clicking **OK, GOT IT**. 
+
+![Out-of-the-box Connectivity Configuration](assets/ootb-wifi-config.png "Out-of-the-box Connectivity Configuration")
+
+***If you face any issue with this flow or prefer to interact directly with your Portenta X8 through the command line, please refer to [this section](#working-with-linux) to learn how to connect with the board leveraging ADB service.***
+
+Once the setup is ready, let's open the Out-of-the-box page. Use the command line window and launch the TCP port forwarding setup command:
```

Comment:
Are we sure we need to do this command? Unfortunately I don't have an X8 with me, but is it not possible to open the out of the box directly after the procedure under board-registration.arduino.cc?

Reply:
I have rechecked this part, and it does transfer the user to the OOTB experience after completing the procedure on board-registration.arduino.cc. However, once the Portenta X8 is disconnected, the OOTB experience is not immediately accessible for post usage and requires running the command. I have updated this segment to clarify the direct transition to OOTB after board registration through the site and the process for accessing OOTB later using the command.